deploy.sh

```diff
@@ -1,27 +1,114 @@
+  # Make it less green
+  echo -e "\033[0m"
+  sleep 1
+fi
+
+# Talk to the user
+echo "Welcome to the single instance Retake installer!"
+echo ""
+echo "Power user or aspiring power user?"
+echo "Check out our docs on deploying Retake! https://docs.getretake.com/deployment/local"
+echo ""
+
+if [ -f ".env" ]; then
+    echo "Environment file .env found, sourcing it."
+    source .env
+else
+    echo "Environment file .env not found. You will be prompted for the following environment variables: RETAKE_APP_TAG, DOMAIN"
+    echo ""
+fi
+
+if ! [ -z "${RETAKE_APP_TAG:-}" ]; then
+    export RETAKE_APP_TAG=$RETAKE_APP_TAG
+else
+    echo "What version of Retake would you like to install? Browse available versions here: https://hub.docker.com/r/retake/retakesearch/tags"
+    read -p "Please enter a valid tag (i.e.: vX.Y.Z) or press Enter to default to 'latest': " RETAKE_APP_TAG
+    if [ -z "$RETAKE_APP_TAG" ]; then
+        export RETAKE_APP_TAG="${RETAKE_APP_TAG:-latest}"
+    else
+        export RETAKE_APP_TAG=$RETAKE_APP_TAG
+    fi
+    echo "Using tag: $RETAKE_APP_TAG"
+fi
+echo ""
+
+if ! [ -z "${DOMAIN:-}" ]; then
+    export DOMAIN=$DOMAIN
+else
+    echo "Let's get the exact domain Retake will be installed on. This will be used for TLS 🔐."
+    echo "Make sure that you have a Host A DNS record pointing to this instance!"
+    read -p "Please enter your configured domain (i.e.: search.getretake.com): " DOMAIN
+    if [ -z "$DOMAIN" ]; then
+        echo "Domain not provided. This step is mandatory, exiting..."
+        exit 1
+    fi
 fi
+export DOMAIN=$DOMAIN
+echo ""
+
+echo "Ok! we'll set up certs for https://$DOMAIN 🎉"
+echo ""
+echo "We will need sudo access so the next question is for you to give us superuser access"
+echo "Please enter your sudo password now:"
+sudo echo ""
+echo "Thanks! 🙏"
+echo ""
+echo "Ok! We'll take it from here 🚀"
+echo ""
 
-DOMAIN=$1
+echo "Making sure any stack that might exist is stopped..."
+sudo -E docker-compose -f docker-compose.yml stop &> /dev/null || true
 
-if [ -n "$2" ]; then
-    echo "Using 'dev' branch as default."
-    GIT_BRANCH="dev"
+# Retake uses basic telemetry to monitor usage (number of deployments, and number
+# of search queries per deployment). If you prefer not to be included in our telemetry,
+# simply set TELEMETRY=disabled in your .env file.
+if ! [ -z "${TELEMETRY:-}" ]; then
+    if [ "${TELEMETRY}" == "disabled"] then;
+        echo "Telemetry successfully disabled -- Retake will not get any usage data from your deployment."
+        echo "Retake has very light telemetry (i.e.: is your deploy running, and how many search queries are you running?)."
+        echo "We do this to get a sense of how much usage Retake is getting, which helps us prioritize features and support."
+        echo "We never collect actual search queries, or PII. If this telemetry is okay with you, please consider re-enabling it."
+        echo "Much love <3!"
```

Comment:
Here you are missing the closing `fi` , otherwise the script won't work.

Reply:
Connect

Reply:
thanks!